elifetools/tests/fixtures/test_authors_json/content_01_expected.py

```diff
@@ -0,0 +1,25 @@
+from collections import OrderedDict
```

Comment:
This file seems minimal, which is important given there are going to be many of this